docs/_docs/plugins.md

```diff
@@ -920,7 +920,7 @@ You can find a few useful plugins at the following locations:
 
 #### Other
 
-- [Analytics for Jekyll](https://github.com/hendrikschneider/jekyll-analytics) by Hendrik Schneider: An effortless way to add  various trackers like Google Analytics, Piwik, etc. to your site
+- [Analytics for Jekyll](https://github.com/hendrikschneider/jekyll-analytics) by Hendrik Schneider: An effortless way to add  various trackers like Google Analytics, Piwik, Matomo, MPulse, etc. to your site
```

Comment:
- Piwik is now Matomo.. so it should not be Piwik, Matomo
- All docs refer to "MPulse" as "mPulse"

```diff
- trackers like Google Analytics, Piwik, Matomo, MPulse, etc. to your site
+ trackers like Google Analytics, Matomo (Piwik), mPulse, etc. to your site
```

Reply:
I think it is also important to mention Piwik to show backwards compatibility. Would you accept this description?

```
An effortless way to add various trackers like Google Analytics, 
Matomo (former Piwik), mPulse, etc. to your site.
```

Reply:
yes.. I'm fine with adding an adjective.. `formerly` sounds more apt though..